firmware/components/driver_framebuffer/driver_framebuffer_window.cpp

```diff
@@ -83,6 +83,10 @@ Window* driver_framebuffer_window_create(const char* name, uint16_t width, uint1
 		window->buffer = (uint8_t*) heap_caps_malloc(((width*height*PIXEL_SIZE)/8)+1, MALLOC_CAP_8BIT);
 	#endif
 	
+	/* Initialise matrix stack */
```

Comment:
I try to comment _why_ I do something rather than _what_ I do, because the what is usually apparent and the why is unclear to others